target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -94,6 +94,17 @@ define Device/tplink_archer-c6-v2
 endef
 TARGET_DEVICES += tplink_archer-c6-v2
 
+define Device/tplink_archer-a6-v2
```

Comment:
Sorting!

Reply:
back to c6-v2-us, so is it fine to keep same sorting?